common/changes/dagaeta-fix-undocumented-stripping-bug_2017-02-07-20-04.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "fixed initialization of ApiFileGenerator._insideTypeLiteral",
```

Comment:
This shows up in the changelog. You should document this as "Fixing issue where undocumented comment was not being emitted."